python/cudf/bindings/cudf_cpp.pxd

```diff
@@ -36,6 +37,7 @@ cpdef check_gdf_error(errcode)
 # mirrors the structure in cpp/include
 
 cdef extern from "cudf.h" nogil:
+cdef extern from "bitmask_ops.hpp" nogil:
```

Comment:
This should be in a separate block than the previous extern, i.e.:
```
cdef extern from "cudf.h" nogil:
    ...

cdef extern from "bitmask_ops.hpp" nogil:
    ...
```

Reply:
Jake resolved this in his PR